ldap/servers/slapd/back-ldbm/back-ldbm.h

```diff
@@ -111,17 +111,19 @@ typedef unsigned short u_int16_t;
  * newidl format.
  * Starting from DS7.2
  */
+#define BE_CHANGELOG_FILE     "replication_changelog"
+
 #define BDB_IMPL              "bdb"
 #define BDB_BACKEND           "libback-ldbm" /* This backend plugin */
 #define BDB_NEWIDL            "newidl"       /* new idl format */
 #define BDB_RDNFORMAT         "rdn-format"   /* Subtree rename enabled */
 #define BDB_RDNFORMAT_VERSION "3"            /* rdn-format version (by default, 0) */
 #define BDB_DNFORMAT          "dn-4514"      /* DN format RFC 4514 compliant */
 #define BDB_DNFORMAT_VERSION  "1"            /* DN format version */
-#define BDB_CL_FILENAME       "changelog.db"
+#define BDB_CL_FILENAME       "replication_changelog.db"
 
 #define LMDB_IMPL             "lmdb"
-#define LMDB_CL_FILENAME      "changelog.mdb"
+#define LMDB_CL_FILENAME      "replication_changelog.mdb"
```

Comment:
several comments on this:
- why change the name "changelog", without automatically update filenames it could break existing deployments
- the bdb/lmdb specific definitions shoulod't be at lmdb level, only known and needed in the impl layer
- the filename for lmdb doesn't make sense, in lmdb there exist only one filename for the backend (or even all backends) and "changelog" is just one database inside this opaque file

Reply:
Well I used "replication_changelog" because I was worried that "changelog" could easily be a custom attribute a customer might use.  Making the name more unique seemed like a safer approach.  Right now there are no releases that have your original changelog work in them, so no one is using changelog.db at the moment.  Figured now was the best time to change it before we get some huge customer that is already using "changelog" as an index and then replication or index breaks.

As for LMDB, I don't know how the database looks and is laid out.  I was tryingto be proactive in this patch :-)   I didn't know the entire database was one file.  There are very little docs out there about such things.  So if it's not needed I can remove that then

Reply:
I agree with the naming, making it more specific is a good thing here.